content/blog/2023-09-27-epidatr.Rmd

```diff
@@ -0,0 +1,94 @@
+---
+title: New package released! `epidatr`, an R client for Delphi Epidata API
```

Comment:
The string "\`epidatr\`" in the title renders literally, which is probably not what was intended.  I'm not sure what to do here.

Reply:
Happy to just get rid of the ticks.